src/applications/edu-benefits/feedback-tool/components/SchoolSelectField.jsx

```diff
@@ -155,7 +154,17 @@ export class SchoolSelectField extends React.Component {
   handleStartOver = e => {
     e.preventDefault();
 
-    this.props.onChange({});
+    this.props.onChange({
```

Comment:
This reseting behavior is unexpected, we plan to investigate a solution, and will link a ticket here once it's set up.